src/mgmt/cli_mgmt.c

```diff
@@ -583,41 +578,37 @@ int dc_mgmt_net_cfg(const char *name)
 
 	/* Allow client env overrides for these three */
 	d_agetenv_str(&crt_timeout, "CRT_TIMEOUT");
-	if (!crt_timeout) {
-		rc = asprintf(&crt_timeout, "%d", info->crt_timeout);
-		if (rc < 0) {
-			crt_timeout = NULL;
-			D_GOTO(cleanup, rc = -DER_NOMEM);
-		}
-		D_INFO("setenv CRT_TIMEOUT=%s\n", crt_timeout);
-		rc = d_setenv("CRT_TIMEOUT", crt_timeout, 1);
-		if (rc != 0)
-			D_GOTO(cleanup, rc = d_errno2der(errno));
-	} else {
+	if (!crt_timeout)
+		crt_info->cio_crt_timeout = info->crt_timeout;
+	else
 		D_DEBUG(DB_MGMT, "Using client provided CRT_TIMEOUT: %s\n", crt_timeout);
```

Comment:
i would explicitly set crt_info->cio_crt_timeout = atoi(crt_timeout); in else case too, or just something like   crt_info->cio_crt_timeout = crt_timeout ? atoi(crt_timeout) : info->crt_timeout. I think its better to initialize cart the same way each time and also will help with print on L611

Reply:
ok i can update that. note that this was actually not changed in that case and left as is.

Reply:
done